src/command_bar.cc

```diff
@@ -38,6 +38,14 @@ namespace Astroid {
         sigc::mem_fun (this, &CommandBar::entry_key_press)
         );
 
+    entry.signal_event ().connect ([&] (GdkEvent* event)->bool {
```

Comment:
Isn't this already covered by the signal connection above?

Reply:
No, at least not on my machine (and it took me a while to figure this out). This event only fires when the text box loses and regains focus (happens for example when pressing the alt key, and I think this is the reason it works for everything else). I wasn't able to find anything in the documentation on this.